Apps/W1/ShopifyConnector/app/Products/Codeunits/ShpfyUpdateItem.Codeunit.al

```diff
@@ -0,0 +1,116 @@
+/// <summary>
+/// Codeunit Shpfy Update Item (ID 30188).
+/// </summary>
+codeunit 30188 "Shpfy Update Item"
+{
+    Access = Internal;
+    TableNo = "Shpfy Variant";
+
```

Comment:
This is correct, because the permissions are already in the role of "D365 BASIC"
Previous we use the permission sets in an xml file, because the permission objects didn't exists.
With the XML file, we must also included these permissions, but now with the permission set objects is this not necessary anymore.

Reply:
Updated with commit: "Inderect Permissions"